ani-cli.1

```diff
@@ -0,0 +1,101 @@
+.\" Automatically generated by Pandoc 2.9.2.1
+.\"
+.TH "ANI-CLI (1) ani-cli" "" "\[lq]June 19 2022\[rq]" "" ""
+.hy
+.SH NAME
+.PP
+ani-cli - watch anime from the commandline
+.SH SYNOPSIS
+.PP
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] [-a ] [-d | -p ] []
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] -c
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] -h | -D | -U | -V
```

Comment:
The whole synopsis is just a mess. 
![image](https://user-images.githubusercontent.com/66648475/208688566-e9dc75ba-7504-4cb7-9988-40b4f8cd0c87.png)
I don't really understand what is it's purpose. We should find it one or if we can't, just reduce it to the minimum